uips/uip-7.md

```diff
@@ -0,0 +1,164 @@
+| uip            | 7                                                                         |
+| -------------- | ------------------------------------------------------------------------- |
+| title          | Noble Forwarding Address Design                                           |
+| description    | Specification of generation of Penumbra-linked Noble forwarding addresses |
+| author         | Chris Czub <chris@penumbralabs.xyz>                                       |
+| discussions-to | <https://forum.penumbra.zone>                                             |
```

Comment:
Forum URL should be https://forum.penumbra.zone/t/pre-uip-noble-address-forwarding/125